### PR TITLE
Use "recovery URL" more consistently.

### PR DESCRIPTION
**Pull request by floitsch**
Also fix output of `recovery export`.

### CODE REVIEW

File: src/cli/cmds/fleet.toit [hunk 1]
```diff
@@ -549,23 +549,23 @@ create-fleet-commands config/Config cache/Cache ui/Ui -> List:
...
+          Add a recovery URL to this fleet.
           """
       --rest=[
         cli.Option "url"
             --help="The URL of the recovery server."
             --required,
       ]
       --examples=[
-        cli.Example "Add a recovery server 'https://recovery.toit.io':"
+        cli.Example "Add a recovery URL 'https://recovery.toit.io':"
```
Comment (kasperl) [inline src/cli/cmds/fleet.toit:560]: This seems like a poor example. For the fleet commands, the URL should probably be more specific (include a fleet id, etc.).

File: src/cli/cmds/fleet.toit [hunk 2]
```diff
@@ -580,7 +580,7 @@ create-fleet-commands config/Config cache/Cache ui/Ui -> List:
             --multi,
       ]
       --examples=[
-        cli.Example "Remove the recovery server 'https://recovery.toit.io':"
+        cli.Example "Remove the recovery URL 'https://recovery.toit.io':"
```
Comment (kasperl) [inline src/cli/cmds/fleet.toit:583]: Here too.

File: tests/gold/cmd-fleet-recovery-test/020-add.txt [hunk 3]
```diff
@@ -1,3 +1,3 @@
 # Add a recovery server
 # [fleet, recovery, add, http://example.com:1234]
-Added recovery server 'http://example.com:1234'.
+Added recovery URL 'http://example.com:1234'.
```
Comment (kasperl) [inline tests/gold/cmd-fleet-recovery-test/020-add.txt:3]: This is kind of a poor example too.

File: tests/gold/cmd-fleet-recovery-test/040-show.txt [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # List the recovery servers
 # [fleet, recovery, list]
-Recovery servers:
+Recovery urls:
```
Comment (kasperl) [inline tests/gold/cmd-fleet-recovery-test/040-show.txt:3]: This is kind of a poor example too.

File: tests/gold/cmd-fleet-recovery-test/050-export.txt [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Export the recovery information
 # [fleet, recovery, export, --directory, TMP_DIR]
 Exported recovery information to 'TMP_DIR/recover--={|~~~~~~~~~~FLEET_ID~~~~~~~~~~|}=-.json'.
-Devices with the current recovery servers configuration will try to
+Devices with the current recovery configuration will try to
   download it from one of the following URLs:
-- http://example.com:1234/recover--={|~~~~~~~~~~FLEET_ID~~~~~~~~~~|}=-.json
-- https://example.com:1234/recover--={|~~~~~~~~~~FLEET_ID~~~~~~~~~~|}=-.json
+- http://example.com:1234
```
Comment (kasperl) [inline tests/gold/cmd-fleet-recovery-test/050-export.txt:6]: This is kind of a poor example too.

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: src/cli/cmds/fleet.toit (hunk 1 above)
Comment (floitsch) [reply src/cli/cmds/fleet.toit]: done.

File: src/cli/cmds/fleet.toit (hunk 2 above)
Comment (floitsch) [reply src/cli/cmds/fleet.toit]: done.

File: tests/gold/cmd-fleet-recovery-test/020-add.txt (hunk 3 above)
Comment (floitsch) [reply tests/gold/cmd-fleet-recovery-test/020-add.txt]: done.

File: tests/gold/cmd-fleet-recovery-test/040-show.txt (hunk 4 above)
Comment (floitsch) [reply tests/gold/cmd-fleet-recovery-test/040-show.txt:3]: done.

File: tests/gold/cmd-fleet-recovery-test/050-export.txt (hunk 5 above)
Comment (floitsch) [reply tests/gold/cmd-fleet-recovery-test/050-export.txt]: done.

**Review by kasperl — APPROVED** [review]
Nice!

**Pull request merged**